### PR TITLE
Add bunch of resource-loading and server-related mappings.

### PR DESCRIPTION
**Pull request by LambdAurora**

### CODE REVIEW

**Review by supersaiyansubtlety — APPROVED** [review]

File: mappings/net/minecraft/server/Main.mapping [hunk 1]
```diff
@@ -1,10 +1,13 @@
 CLASS net/minecraft/server/Main
 	FIELD f_raofjkac LOGGER Lorg/slf4j/Logger;
+	METHOD m_cbzunkqe (Lnet/minecraft/unmapped/C_bpwagnit$C_unecclfw;Ljoptsimple/OptionSet;Ljoptsimple/OptionSpec;Lnet/minecraft/unmapped/C_veyxvrwr;Ljoptsimple/OptionSpec;Lnet/minecraft/unmapped/C_tmnrpasf;Lnet/minecraft/unmapped/C_tjllhmjy;)Lcom/mojang/datafixers/util/Pair;
```
Comment (IotaBread) [inline mappings/net/minecraft/server/Main.mapping]: Why is this line here?

**Review by IotaBread — APPROVED** [review]

File: mappings/net/minecraft/server/Main.mapping (hunk 1 above)
Comment (LambdAurora) [reply mappings/net/minecraft/server/Main.mapping]: I have no idea, ask Enigma!

**Review by NoComment1105 — APPROVED** [review]
